pkg/releaselock/releaselock.go

```diff
@@ -0,0 +1,89 @@
+package releaselock
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"net"
+	"net/rpc"
+	"strconv"
+
+	"github.com/prashanthpai/sunrpc"
+	"k8s.io/klog/v2"
+)
+
+const (
+	pmapProgramNumber  = uint32(100021)
+	pmapProgramVersion = uint32(4)
+	programNumber      = uint32(200002)
+	programVersion     = uint32(1)
+	procedureNumber    = uint32(1)
+	procedureName      = "IN_BAND_PROPRIETARY_LOCK_OPS_PROG.IN_BAND_PROPRIETARY_LOCK_OPS_V1"
```

Comment:
Just to be clear, this is not the procedure name. `IN_BAND_PROPRIETARY_LOCK_OPS_PROG` is the name of the program, and `IN_BAND_PROPRIETARY_LOCK_OPS_V1` is the name of the version.  The procedure name is `RELEASE_ALL_LOCKS`.

The name doesn't matter for the RPC, but I would change it for clerity.

The procedure name is require to be of the form 'T.MethodName' (see https://github.com/prashanthpai/sunrpc/blob/689a3880d90a71848bd2ec8a0f414e208ebc5d90/procedure_registry.go#L52), so I would call it IN_BAND_PROPRIETARY_LOCK_OPS_PROG.RELEASE_ALL_LOCKS

Reply:
Done